src/configuration/routes/cache.md

```diff
@@ -23,58 +42,80 @@ https://{default}/:
         default_ttl: 60
 ```
 
-In this example, requests will be cached based on the URI, the `Accept` header, `Accept-Language` header, and `X-Language-Locale` header; Any response that lacks a `Cache-Control` header will be cached for 60 seconds; and the presence of any cookie in the response will disable caching of that response.
+## Cache configuration properties
 
-## Cache Per Route
+### `enabled`
+
+Turns the cache on or off for a route.
 
-If you need fine-grained caching, you can set up caching rules for
-several routes separately.
+**Type:** Boolean
 
-Here is an example:
+**Values**
+* `true`: enable the cache for this route
+* `false`: disable the cache for this route [default]
```

Comment:
There is no default here: `enabled` is a required key if you have `cache` (and if you don't have `cache`, the cache will be enabled)